tests/cases/transform.js

```diff
@@ -165,8 +168,11 @@ describe('geo.transform', function () {
         results: []
       }));
 
-      expect(spy.calledOnce).toBe(true);
-      expect(geo.transform.defs.hasOwnProperty('EPSG:5001')).toBe(false);
+      window.setTimeout(function () { // wait for the next time slice
```

Comment:
why we need window timeout now?

Reply:
jQuery 3 is always asynchronous for certain requests where jQuery 2 was sometimes synchronous. Specifically, we are making an ajax request to get the transform from a web service.  We mock the web service for the test, so the mock returns immediately.  But, in jQuery 3, the callback is fired in the next time slice rather than synchronously, so we wait 0 ms (just to be in the next time slice) to make sure the callback has been triggered. 

In jQuery 2, the behavior was ajax->mock response->callback->next time slice.

In jQuery 3, the behavior is ajax->mock response->next time slice->callback.

If we were doing an actual request which couldn't be served from browser cache, we would have to wait an unknown amount of time before the callback would be triggered.  It is only because we are mocking the response that the immediate check worked in jQuery 2 (and the check with a 0-duration wait works in jQuery 3).  Rather than use window.setTimeout, we could have used time mocking, but that doesn't make the test any clearer in my opinion.

Reply:
thanks for the detailed explanation @manthey.